### ISSUE TITLE
EnderIO materials don't respect any changes

### ISSUE THREAD

**Issue by ReineRosette** [RCXcrafter/TweakersConstruct#19]
As the title says, disabling EnderIO materials does not work, and changing the stats also fails to do anything.

**Comment by 0ptera** [comment]
I'm having the same issue with Enderio. Could you please direct me to a release including this fix.

The latest release I found was 1.12.2-1.5.1 on curse which is almost a month older than this fix.

**Comment by Normal69** [comment]
Good news everyone!
I could repoen this issue with some more information on the topic.

I've used your settings to load default values from the game - which is a great idea and worked perfectly.
B:"Fill Defaults Traits"=true

It perfectly worked like this:
        fierymetal:head:twilit,autosmelt,superheat,flammable
        fierymetal:projectile:twilit,flammable
        fierymetal:handle:twilit,flammable
        fierymetal:extra:twilit,flammable
        fierymetal:bow:twilit,flammable
        fierymetal:shaft:twilit,flammable

Then I've removed some materials, and moved around the traits and values.
Now, several materials are missing their traits, the values are correct.

Looking into the logs I saw several traits giving error like this:
[21:39:15] [Client thread/WARN] [tweakersconstruct]: [Trait Tweaks] Could not find trait: flammable, skipping.
So the trait value your mod has read from the game is not recognised when writing it back.

The problem appears with different several traits from Twilightforest, Construct Armory, et cetera.
I just append the part from the logfile, hope it helps: https://pastebin.com/NQ7CQB3G
The original config after running the load default options: https://pastebin.com/p0Lyye8p
My modified config which throws the log problems: https://pastebin.com/WpwHxWgJ

Also notify me if I can provide more data to help you to discover the cause!
I would like to have this fine mod made perfect. :)
